### DISCUSSION TITLE
buddhism/2021/docs_2021/sadhana/urgyen/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#144] [General]
# buddhism/2021/docs_2021/sadhana/urgyen/

http://shanenull.com/buddhism/2021/docs_2021/sadhana/urgyen/

**Comment by shane0** [comment]
https://youtu.be/EwqomX3mwmA?si=SDzc0sQIX5EUMJcl

**Reply by shane0** [reply]
Shortened remaster
https://youtu.be/5QYJ5mVrm7E?si=tDciS5qf7tiFHeH6
